opencga-app/src/main/java/org/opencb/opencga/app/cli/main/options/JobsCommandOptions.java

```diff
@@ -192,8 +194,8 @@ public class DistinctCommandOptions {
         @Parameter(names = {"--user-id"}, description = "User that created the job", required = false, arity = 1)
         public String userId; 
     
-        @Parameter(names = {"--priority"}, description = "Priority of the job", required = false, arity = 1)
-        public String priority; 
+        @Parameter(names = {"--job-priority"}, description = "Priority of the job", required = false, arity = 1)
```

Comment:
This should be "priority", not "job-priority", as we're already within jobs

Reply:
Fixed